main/src/ca/uwaterloo/flix/language/phase/jvm/GenSimpleLocationErrorClass.scala

```diff
@@ -0,0 +1,147 @@
+/*
```

Comment:
Why is this class called GenSimpleLocationError? Whats the relationship between this and Reified?

Reply:
The context is that the MatchError.class and NotImplementedError.class were identical except a single string given to the super constructor, so I didn't want to copy everything in Gen[Match/Hole]ErrorClass. The GenSimpleLocationError name was made to describe that its a simple subclass of FlixError with a prefix string and a location.

Reply:
Ah- Could we call it something better? `GenMatchOrHoleError`? or other suggestions welcome. Maybe just `GenFlixErrorSubClass`?

I thought it was generating a *location* thingy, but that's not the case I now understand.